ovos_classifiers/heuristics/normalize.py

```diff
@@ -230,9 +230,22 @@ class GermanNormalizer(Normalizer):
 
     def numbers_to_digits(self, utterance: str) -> str:
         return GermanNumberParser().convert_words_to_numbers(utterance)
-    
+
     def remove_symbols(self, utterance: str) -> str:
         # special rule for hyphanated words in german as some STT engines falsely
         # return them pretty regularly
         utterance = re.sub(r'\b(\w*)-(\w*)\b', r'\1 \2', utterance)
         return super().remove_symbols(utterance)
+
+class FrenchNormalizer(Normalizer):
+    with open(f"{dirname(dirname(__file__))}/res/fr/normalize.json") as f:
+        _default_config = json.load(f)
+
+    def remove_symbols(self, utterance: str) -> str:
+        # special rule for hyphanated words in french as some STT engines falsely
+        # return them pretty regularly
+        utterance = re.sub(r'\b(\w*)-(\w*)\b', r'\1 \2', utterance)
+        return super().remove_symbols(utterance)
```

Comment:
maybe add
```
def remove_articles(self, utterance):
        words = self.tokenize(utterance)
        for idx, word in enumerate(words):
            if word.startswith("l'") or word.startswith("d'"):
                words[idx] = word[2:]
            elif word in self.articles:
                words[idx] = ""

        return " ".join([w for w in words if w != ""])
```

those contraction d'/l' can't be caught by the decontract mechanism, since it would need a pretty extensive table